payments-core/src/main/java/com/stripe/android/payments/core/injection/StripeRepositoryModule.kt

```diff
@@ -27,4 +34,23 @@ abstract class StripeRepositoryModule {
     internal abstract fun bindsStripeRepository(
         stripeApiRepository: StripeApiRepository
     ): StripeRepository
+
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
Why not `internal`?

Reply:
https://github.com/stripe/stripe-android/pull/10550/commits/ba0e5459a91655a44d6b4eb5e5469007814613f7